wallet-new/spec/swagger.json

```diff
@@ -1036,10 +1036,6 @@
                         "description": "Invalid `new` or `old`"
                     },
                     "200": {
-                        "schema": {
-                            "items": [],
-                            "type": "array"
-                        },
```

Comment:
Btw am I right that swagger doc will be wiped out from git eventually?

Reply:
No, they will stay 😉  The main point of having them in git is that we can always have a persistent remote link where we can point people at.

for example, we can already say:

"Hey, go to `editor.swagger.io` , click `File -> Import URL` and then paste the following:

https://raw.githubusercontent.com/input-output-hk/cardano-sl/5aeca683cd0bf32cdd9d92aaad6427acca42aaa0/wallet-new/spec/swagger.json

"

What would be quite a boon as they would be able to fetch the exact snapshot of the API at each stage of the development, without relying on things like https://cardanodocs.com/technical/wallet/api/ which can very easily get obsolete/outdated. "